skill-tree /src/test/java/com/RDS/skilltree/TestContainerManger.java

```diff
@@ -0,0 +1,24 @@
+package com.RDS.skilltree;
+
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MySQLContainer;
+
+abstract class TestContainerManger {
+    @ServiceConnection
+    static final MySQLContainer mysqlContainer;
+
+    static {
+        mysqlContainer = new MySQLContainer("mysql:latest")
+                .withDatabaseName("skilltree-test")
+                .withUsername("root")
+                .withPassword("password");
+        mysqlContainer.start();
```

Comment:
Container starts up here. Where is the stop hook?

Reply:
So this has been done, to use the same container for all the test files and using the same container for all test files, it will stop when all the tests are completed.
I've got to know from the Test container blog https://java.testcontainers.org/test_framework_integration/manual_lifecycle_control/#singleton-containers